### PR TITLE
Bugfix/file references

### PR DESCRIPTION
**Pull request by fred-perkins**

### CODE REVIEW

File: src/DSharp.Sdk/Sdk/Pack.targets [hunk 1]
```diff
@@ -61,7 +69,7 @@
       <TfmOutputDirs Include="$(OutputPath)">
         <TargetFramework>$(TargetFramework)</TargetFramework>
         <TargetFrameworkIdentifier>$(TargetFrameworkIdentifier)</TargetFrameworkIdentifier>
-        <PackagePath>lib\$(TargetFramework)\</PackagePath>
+        <PackagePath>contentFiles\any\$(TargetFramework)\</PackagePath>
```
Comment (isc30) [inline src/DSharp.Sdk/Sdk/Pack.targets]: are we sure about this change? it seems to break the library DLLs for both targets
- dsharp target is generating a dll next to the js that will be missplaced now
- netstandard output will always be missplaced

Comment (fred-perkins) [reply src/DSharp.Sdk/Sdk/Pack.targets]: @KevReed Can you review Ivans comment?

Comment (KevReed) [reply src/DSharp.Sdk/Sdk/Pack.targets]: It seems that when using PackageReference format content in the lib folder is not copied alongside the dlls. This change is to update the location within the package of .js files output by the transpiler. Within the package they now live in contentFiles, but the nuspec generated adds metadata to copy to the correct bin folder when consuming the package (this is done by adding these files as linked static assets in the Content itemgroup with CopyToOutput set). these .js files will only be available/relevant for net framework builds (i.e. transpiled builds).

Comment (KevReed) [reply src/DSharp.Sdk/Sdk/Pack.targets]: If we think this needs to be more descriptive I can update this metadata to be called PackageContentPath or similar, which is mapped to the actual itemgroup at a later stage (see https://github.com/DerivcoIpswich/dsharp/pull/150/files/7cbf242215cb67434403c7d1bf604bc5b5d5ec88#diff-190d96f9354681bae9679b2cd4c9b748R60).

Comment (isc30) [reply src/DSharp.Sdk/Sdk/Pack.targets]: yeah, that sounds like a good solution

**Comment by fred-perkins** [comment]
Before I Merge I just want to run a build on TC which collects these artifacts - Currently it doesn't pick up the new script nupkg

**Pull request merged**